backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -291,7 +292,7 @@ def execute_write_query(
             f"INSERT INTO {table_name} ({','.join(sql_keys)}) "
             f"SELECT {','.join(sql_values)}"
         )
-        logger.debug(f"insertng into table with: {sql} query")
```

Comment:
is it ok to print this as info level? As the logs contain actual values that are getting inserted!

Reply:
I am making this as debug

Reply:
Made it as logger.debug @johnyrahul